### PR TITLE
CON-251: connector js release notes for 6.1.1/1.2.0

### PR DESCRIPTION
**Pull request by rkorte**
Make sure I put the new features/fixes in the right product (e.g., CON-209 is python only)

Will do these in separate PRs coming up:
- CON-234 (fix broken link)
- CON-252 (links to core docs should use release number, not 'current')
- Should we send copyright_license.rst page to Legal for review? Or do you know for a fact that hasn't changed?

### CODE REVIEW

File: docs/release_notes.rst [hunk 1]
```diff
@@ -42,24 +42,67 @@ New Platforms
...
+
 Previous Releases
-~~~~~~~~~~~~~~~~~
+-----------------
+
+Version 1.1.1
+^^^^^^^^^^^^^
+*Connector* 1.1.1 is built on *RTI Connext DDS* 6.1.0.3, which fixes several
+bugs in the Core Libraries. If you want more details on the bugs fixed in 6.1.0.3,
+contact support@rti.com. These bugs are also fixed in
```
Comment (rkorte) [inline docs/release_notes.rst:84]: I added this last sentence. It certainly explains things for me. But will customers be confused? Let's get @alexcamposruiz to weigh in, too.

File: docs/release_notes.rst [hunk 2]
```diff
@@ -42,24 +42,67 @@ New Platforms
 
 RTI has validated that *Connector* can be used on macOS 11 (Big Sur) systems.
 
+
+New API makes it easier to query what version of Connector is being used
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. CON-92 
+
+A new API has been added that provides the caller with the version of *Connector*
+and the version of the native libraries being used.
```
Comment (rkorte) [inline docs/release_notes.rst:51]: @samuelraeburn, where should I link to for information about the API? Please feel free to update the branch directly.

File: docs/data.rst [hunk 3]
```diff
@@ -191,6 +191,9 @@ getter: :meth:`SampleIterator.getNumber()`, :meth:`SampleIterator.getBoolean()`,
     a value that can be interpreted as a number, ``sample.get('my_string')`` returns
     a number, not a string.
 
+
+.. _section-access-64-bit-integers-js:
+
 Accessing 64-bit integers
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 Internally, *Connector* relies on a framework that only contains a single number
```
Comment (rkorte) [inline docs/data.rst:199]: @samuelraeburn, remember in this file somewhere, you were going to remove a Warning and add it to somewhere in the release notes: https://github.com/rticommunity/rticonnextdds-connector-py/pull/102#discussion_r751983361. Feel free to update this branch directly. (Or did that apply only to py?)

File: docs/release_notes.rst (hunk 2 above)
Comment (samuelraeburn) [reply docs/release_notes.rst:51]: Added

File: docs/data.rst (hunk 3 above)
Comment (samuelraeburn) [reply docs/data.rst:199]: added

File: docs/release_notes.rst (hunk 1 above)
Comment (samuelraeburn) [reply docs/release_notes.rst:84]: I think it makes sense. Let's see what @alexcamposruiz says

**Comment by samuelraeburn** [comment]
All looks good to me. Yes I think we should get them to check the License file again @rkorte

**Review by samuelraeburn — APPROVED** [review]

**Pull request merged**